docs/source/installation/production/debian/nginx.rst

```diff
@@ -38,6 +38,7 @@ Afterwards, make sure the services you just installed are running:
 .. code-block:: shell
 
     systemctl start postgresql.service redis-server.service
+    pg_ctlcluster 13 main start
```

Comment:
I don't see why this would be needed - `systemctl start postgresql.service` should start it, no?

Reply:
It doesn't in my case. I'm running this in a ubuntu:22.04 Docker container